Dict_practice.py

```diff
@@ -0,0 +1,9 @@
+# WAP to alter the value of a key passed by user and print the modified dictionary
+def main():
+    Dict_1 = {"Name":"Anuva","age":25}
+    User_input_value = int(input("Enter the age value to be updated \n"))
```

Comment:
you can ignore this for now but start using naming convention like either snake case or kabka etc and be uniform across the program

Reply:
OK sure